Form.css

```diff
@@ -0,0 +1,22 @@
+.Form {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.Form * {
+  margin-bottom: 30px;
+}
+
+.Form input {
+  width: 10%;
+  transition: width 2s;
+}
+
+.Form input:focus {
+  width: 100%;
+}
+
+.no-padding {
```

Comment:
This type of selector (that describe what the class _does_ rather than what it _is_) violates semantic naming conventions. Moreover, the need for a `no-padding` class seems to stem from the questionable use of the universal selector above. I would remove this class entirely in favour of a more specific selector that only targets the `inputs` and doesn't _accidentally_ effect the final `button`.